static/app/components/platformPicker.tsx

```diff
@@ -100,12 +102,12 @@ class PlatformPicker extends Component<PlatformPickerProps, State> {
 
   render() {
     const platformList = this.platformList;
-    const {setPlatform, listProps, listClassName} = this.props;
+    const {setPlatform, listProps, listClassName, modal} = this.props;
```

Comment:
I don't love `modal` as a prop since it's only usable for this one specific use case. Would rather pass in some more descriptive prop names to customize it for the new use case. Ideally we would split this into multiple components and you could compose it as you see fit, but that would be more work that we need right now. So. here's some simpler suggestions:

- Add a `showFIlterBar` prop that defaults to true
- Use the existing `listClassName` prop to restyle PlatformList
- Add a new `navClassName` prop to restyle NavContainer